src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+
+public class Formatter {
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    
```

Comment:
Missing data members such as `LINE_PREFIX`, `DIVIDER` and some more. Think how the absence of those members violates SRP.